recipe-client-addon/content/about-studies/shield-studies.js

```diff
@@ -78,12 +78,7 @@ class StudyList extends React.Component {
   }
 
   receiveRemoteValue(name, value) {
-    // Convert date strings to dates.
-    const studies = value.map(study => {
-      study.studyStartDate = stringToDate(study.studyStartDate);
```

Comment:
IndexedDB can store Date objects; we were stringifying them before because the JSONSchema validation didn't handle date objects (which really indicated that it wasn't quite the right fit for our use case, but I didn't want to have two different schema formats in our codebase).

Since we're no longer accepting the entire object to be stored in IndexedDB, we don't really need full schema validation, and thus don't need to stringify the dates.